src/components/Navbar/index.tsx

```diff
@@ -36,6 +39,24 @@ export default function Navbar() {
     const handleOpenSidebar = () => {
         setOpenSidebar(true)
     }
+    const dispatch = useAppDispatch()
+    useEffect(() => {
+        let timer
+        if (auth && window.location.hostname !== 'localhost')
+            timer = new IdleTimer({
+                timeout: TIMEOUT_DURATION,
+                onTimeout: async () => {
+                    await store.dispatch('logout')
+                    dispatch(updateAccount(null))
+                    dispatch(updateAuthStatus(false))
+                },
```

Comment:
Add notification here.

Reply:
notifications will be refactored check TECH-712